docs/experiments-msmarco-passage.md

```diff
@@ -460,4 +460,5 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@Yuan-Hou](https://github.com/Yuan-Hou) on 2024-01-02 (commit [`1ebe6dd`](https://github.com/castorini/anserini/commit/1ebe6dda261218b5a1481dcfeaeb47cfc83d0292))
 + Results reproduced by [@himasheth](https://github.com/himasheth) on 2024-01-08 (commit [`1ebe6dd`](https://github.com/castorini/anserini/commit/1ebe6dda261218b5a1481dcfeaeb47cfc83d0292))
 + Results reproduced by [@Tanngent](https://github.com/Tanngent) on 2024-01-13 (commit [`ebde6bc`](https://github.com/castorini/anserini/commit/ebde6bc3faaf0406d5441d2ff480ba3efc43b9e8))
++ Results reproduced by [@Gavin](https://github.com/BeginningGradeMaker) on 2024-01-15 (commit [`85fad8d`](https://github.com/BeginningGradeMaker/anserini/commit/85fad8d9a3aa797b12362894e40aa00041bf3664))
```

Comment:
Please replace `Gavin` with your actual GitHub userid.

Comment should be a commit on Anserini trunk, i.e., `https://github.com/castorini/anserini/` not `https://github.com/BeginningGradeMaker` - make sure it's a valid commit on trunk.